### PR TITLE
[argocd-config] Fix syncwaves.

### PR DESCRIPTION
**Pull request by kfyharukz**
Signed-off-by: Kazuhito MATSUDA <kazuhito.matuda@gmail.com>

### CODE REVIEW

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

File: test/validation_test.go [hunk 1]
```diff
@@ -79,6 +78,7 @@ func testApplicationResources(t *testing.T) {
 		"monitoring":           "7",
 		"sandbox":              "7",
 		"teleport":             "7",
+		"pvc-autoresizer":      "8",
 		"argocd-ingress":       "9",
```
Comment (ymmt2005) [inline test/validation_test.go:82]: These should be decremented because they do not depend on `pvc-autoresizer`.

**Review by ymmt2005 — DISMISSED** [review]
LGTM

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**